packages/react/src/components/CardEditor/CardEditForm/CardEditFormItems/DataSeriesFormItemModal.jsx

```diff
@@ -173,8 +173,10 @@ const defaultProps = {
     dataItemSource: 'Data item source',
     primaryButtonLabelText: 'Save',
     secondaryButtonLabelText: 'Cancel',
-    decimalPlaces: 'Decimal places',
+    decimalPlacesLabel: 'Decimal places',
```

Comment:
Should follow the convention of the others. 
```suggestion
    decimalPlacesLabelText: 'Decimal places',
```